scripts/run_e2e_tests.py

```diff
@@ -419,10 +419,10 @@ def start_google_app_engine_server(dev_mode_setting, log_level):
     app_yaml_filepath = 'app%s.yaml' % ('_dev' if dev_mode_setting else '')
 
     p = subprocess.Popen(
-        '%s %s/dev_appserver.py --host 0.0.0.0 --port %s '
+        'yes | %s %s/dev_appserver.py --host 0.0.0.0 --port %s '
         '--clear_datastore=yes --dev_appserver_log_level=%s '
         '--log_level=%s --skip_sdk_update_check=true %s' % (
-            common.CURRENT_PYTHON_BIN, common.GOOGLE_APP_ENGINE_HOME,
```

Comment:
Is GOOGLE_APP_ENGINE_HOME still used anywhere? If not, we might want to look into getting rid of it altogether.

Reply:
I checked this already and it turns out we use one of the libraries, namely webob_0_9 here in setup_gae.py.

`sys.path.append(
        os.path.join(common.GOOGLE_APP_ENGINE_HOME, 'lib', 'webob_0_9'))`

Reply:
Do we still need setup_gae.py? That line existed before google-cloud-sdk was a thing, so might be worth checking if it's still necessary.

Reply:
Sorry -- I meant the parts of setup_gae that relate to installing google_appengine. (We may want to rename the file to setup_gcloud_sdk, too.)

Reply:
Hm so currently as it turns out the entire codebase uses a lot of google_app engine still. Not only is webOb a package, we also use jinja2, yaml, and webapp2 from the lib folder. We also use the dev_appserver to fix the system path, (a function that no longer exists in the newer version of dev_appserver). 

For some reason we also add the google_appengine home directory into the system path which is also necessary and I'm not even sure what functionality we need from the home directory. 
Should I make a migration to remove all of those dependencies and install the ones we need in oppia_tools after I fix the e2e tests?

Reply:
Yep fixed this.